app/views/idv/how_to_verify/show.html.erb

```diff
@@ -37,6 +37,7 @@
           @idv_how_to_verify_form,
           html: {
             id: nil,
+            class: @selfie_required ? 'margin-top-3 tablet:margin-top-4' : 'margin-top-3 tablet:margin-top-3 ',
```

Comment:
Why do we want different spacing depending if selfie is required or not?

Reply:
Selfie has an additional tag. I see if I can rework some of this to not have to use the conditional.

Reply:
I was confused about this as well but figured it was the tag

Reply:
Resolved with commit 0c73d5a420d729944dece496a100ee0a14f23527 
I will attach new screenshots of view.